### PR TITLE
Added info message with form and telegram link to dashboard

### PR DESCRIPTION
**Pull request by GabriFila**
# Description

This PR adds a little section to the dashboard with the infos about the telegram channel. 

A preview

newLook

### CODE REVIEW

File: dashboard/views/crownLabs/CrownLabsHome.js [hunk 1]
```diff
@@ -142,6 +142,53 @@ function CrownLabsHome(props) {
...
+      notification.open({
+        message: '',
+        description: (
+          <div
+            style={{
+              textAlign: 'center',
+              margin: 20
+            }}
+          >
+            We would like to know you thoughs about CrownLabs! Please fill{' '}
```
Comment (giorio94) [inline dashboard/views/crownLabs/CrownLabsHome.js:159]:
```suggestion
            We would like to know your thoughts about CrownLabs! Please fill{' '}
```

**Review by giorio94 — APPROVED** [review]
/lgtm

Just a couple of linguistic suggestions.
And a curiosity. I remember you showed another alternative with the border around the popup instead of around the text, which I felt more intuitive. Wasn't that working?

**Review by palexster — APPROVED** [review]

**Pull request merged**